CONTRIBUTING.md

```diff
@@ -159,7 +159,7 @@ NOTE: Partially copied from https://raw.githubusercontent.com/emberjs/ember.js/m
 
 Have you got enough knowledge in a specific feature and want to help with docs?
 Ember-cli documentation lives at the branch
```

Comment:
sorry, my statement looked like a question, but was really just asking if you should change the word "branch" to "repository" on that line.